src/contrib/serializers/Akka.Serialization.Hyperion/HyperionSerializerSetup.cs

```diff
@@ -10,31 +10,50 @@
 using Akka.Actor;
 using Akka.Actor.Setup;
 using Akka.Util;
+using Hyperion;
 
 namespace Akka.Serialization.Hyperion
 {
     public class HyperionSerializerSetup : Setup
     {
         public static readonly HyperionSerializerSetup Empty =
-            new HyperionSerializerSetup(Option<bool>.None, Option<bool>.None);
+            new HyperionSerializerSetup(Option<bool>.None, Option<bool>.None, null, null, null);
 
+        [Obsolete]
+        public static HyperionSerializerSetup Create(
+            bool preserveObjectReferences, 
+            bool versionTolerance, 
+            Type knownTypesProvider)
+            => new HyperionSerializerSetup(preserveObjectReferences, versionTolerance, knownTypesProvider, null, null);
+
+        [Obsolete]
         public static HyperionSerializerSetup Create(
             bool preserveObjectReferences, 
             bool versionTolerance, 
             Type knownTypesProvider, 
             IEnumerable<Func<string, string>> packageNameOverrides)
-            => new HyperionSerializerSetup(preserveObjectReferences, versionTolerance, knownTypesProvider, packageNameOverrides);
+            => new HyperionSerializerSetup(preserveObjectReferences, versionTolerance, knownTypesProvider, packageNameOverrides, null);
 
+        public static HyperionSerializerSetup Create(
```

Comment:
Do you want to provide an overload here that takes fewer arguments?

Reply:
The other `Create` static methods takes fewer arguments, but they're all marked as obsolete for backward compatibility reason and signalling the user that there is a new version of the method available.
I can't set default values for the most complete one because the compiler will complain that the method pattern will match the obsolete ones.
There is no danger with using the older methods, but you're right, it doesn't really convey that well.
Do you have any suggestion on how to deal with this?

Reply:
In that case, maybe let's not make them obsolete - treat the more "advanced" features as opt-in choices for users who need them. Have the older `Create` methods with fewer constructor arguments call the new `Create` method and provide some sensible defaults (i.e. `null`) for the new arguments.

Reply:
Done.